### PR TITLE
Add unittest for loopback device

### PR DESCRIPTION
**Pull request by yadavpooja**
We solved below problem 
- https://gist.github.com/yadavpooja/17e041464b54306e324b62e3b995f56e

### CODE REVIEW

**Comment by kushaldas** [comment]
```
command: sudo python3 -m unittest tunirtests.loopbacktest.Loopbacktest -v
status: False

loopbacktest (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: loopbacktest (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: loopbacktest
Traceback (most recent call last):
  File "/usr/lib64/python3.5/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/var/home/fedora/tunirtests/loopbacktest.py", line 2, in <module>
    from testutils import system
ImportError: No module named 'testutils'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```

After fixing this locally, I can see the below

```
command: sudo python3 -m unittest tunirtests.loopbacktest.Loopbacktest -v
status: False

test_loopback (tunirtests.loopbacktest.Loopbacktest) ... ERROR

======================================================================
ERROR: test_loopback (tunirtests.loopbacktest.Loopbacktest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/home/fedora/tunirtests/loopbacktest.py", line 27, in test_loopback
    out = out.encode('utf8')
AttributeError: 'bytes' object has no attribute 'encode'

----------------------------------------------------------------------
Ran 1 test in 0.095s

FAILED (errors=1)
```

Have you tested your code before submitting?

**Comment by yadavpooja** [comment]
Issue was because of using `encode` method instead `decode` and it's fixed now. Please give it another try.

File: loopbacktest.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import unittest
+from testutils import system
```
Comment (trishnaguha) [inline loopbacktest.py]: @yadavpooja We are doing relative import. Can you please change this to `from .testutils import system`?

Comment (praveenkumar) [inline loopbacktest.py]: @trishnaguha This is now fixed, please give it a try.

**Comment by trishnaguha** [comment]
LGTM! Thanks @yadavpooja

**Pull request merged**